src/features/commands/watch.js

```diff
@@ -73,6 +73,13 @@ module.exports = {
                         })
                         return;
                     }
+                    if (keyword.length >= 100) {
+                        interaction.reply({
+                            content: `Keyword cannot be longer than 100 characters.`,
```

Comment:
Good change, albeit unrelated to poll feature. My suggestion is to change this to a simple string as this won't have any dynamic content.
```suggestion
                            content: 'Keyword cannot be longer than 100 characters.',
```